README.md

```diff
@@ -23,11 +24,16 @@ vim .env
 ### Running
 ```bash
 set -a && source .env
+python start.py -r
+# or
 python -m app.main
 # or
 uvicorn --reload app.main:app --log-level=info
 ```
 
+It is assumed that you've configured your .env properly. This includes the expectation that a Postgres
+instance, Gitea Assist, message broker, and celery worker(s) are all configured and running.
```

Comment:
Any message broker works or specifically redis? `assemble_broker_uri` in config.py are creating redis specific URIs so not sure

Reply:
Yeah, I'll update the text. We're going to go with strictly Redis.